ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -101,6 +105,7 @@ function ecms_preprocess_paragraph__media_item(array &$variables): void {
     $image = $paragraph->get('field_media_item')->entity->get('field_media_item_image');
 
     if (isset($image->entity->uri)) {
+      // @todo Is $variables['image_url'] being used? Not seeing it in twig.
       $variables['image_url'] = file_create_url($image->entity->uri->value);
     }
```

Comment:
This can be removed. We switched this when we used responsive image styles and forgot to remove it. We can do it in this PR or another

Reply:
Took care of it here, thanks!